ldservices/client_sdk_data.go

```diff
@@ -1,126 +0,0 @@
-package ldservices
-
-import (
-	"encoding/json"
-
-	"gopkg.in/launchdarkly/go-sdk-common.v1/ldvalue"
-)
-
```

Comment:
This file wasn't actually being used. If I come up with a use case for it, I'll re-add it and possibly rethink the design.